packages/flutter/lib/src/animation/animations.dart

```diff
@@ -384,7 +384,7 @@ class CurvedAnimation extends Animation<double> with AnimationWithParentMixin<do
   final Animation<double> parent;
 
   /// The curve to use in the forward direction.
-  Curve curve;
+  Curve? curve;
```

Comment:
🤷‍♂️

Reply:
its pretty clear that this is the wrong choice. It increasingly seems like null_assertions are not landable

Reply:
Why nullable? From what I can tell this shouldn't be null?

Reply:
(See also constructor and comment there)